internal/pgadmin/users.go

```diff
@@ -72,11 +74,47 @@ if sys.path[0] != root:
 	//
 	// TODO(cbandy): pgAdmin v4.21 adds "auth_source" and "username" as required attributes.
 	// - https://git.postgresql.org/gitweb/?p=pgadmin4.git;f=web/pgadmin/model/__init__.py;hb=REL-4_21#l65
+
+	// When the users are created or modified, server groups and connections will
+	// also be configured, similar to the way server groups and connections are
+	// made with their respective dialog windows.
+	// - https://www.pgadmin.org/docs/pgadmin4/latest/server_group_dialog.html
+	// - https://www.pgadmin.org/docs/pgadmin4/latest/server_dialog.html
+
+	// We use a similar method to the import method when creating server connections
+	// - https://www.pgadmin.org/docs/pgadmin4/development/import_export_servers.html
+	// - https://git.postgresql.org/gitweb/?p=pgadmin4.git;a=blob;f=web/setup.py;h=eb7d9044521a65781e6bdfebf02c8b2572f1dc97;hb=REL-4_20#l256
+
+	// One server connection will be configured for each defined user.
+	// The name of the server connection will be the same as the cluster name.
+	// Note that the server connections are created when the users are created or
+	// modified. Changes to a server connection will generally persist until a
+	// change is made to the corresponding user. For custom server connections,
```

Comment:
🤔 Do we reconcile all users when something in `spec.users` changes? I'm not quite following what this sentence is about: "Changes to a server connection... corresponding user."

Reply:
Yes, if something in `spec.users` changes, we will reconcile users. What I'm trying to call out is that certain changes to the server connection itself, made via the pgAdmin UI, will remain in place until a reconcile takes place. 

For example, if the user should remove the server connection via the UI, it won't be recreated until something that we manage, such as a user password or postgres port, is changed and reconciled.

There may be a way to trigger an update to the server connection so that this happens more often, but I haven't been able to determine how to do so.